packages/@okta/vuepress-site/docs/guides/mobile-idx-sdk-overview/main/index.md

```diff
@@ -0,0 +1,158 @@
+---
+title: Overview of the mobile Identity Engine SDK
+---
+
+<div class="oie-embedded-sdk">
+
+Design the implementation of the sign-in flow for your mobile app by understanding the
+representation of the flow, and with examples of the common parts.
+
+<ApiLifecycle access="ie" /><br>
+
+## Introduction
+
+Integrate the Okta sign-in flow into your app in one of two ways: redirect the request to
+Okta which handles the flow in its own window, or handle each step in the flow by
+building the appropriate user interface elements, capturing the information, and updating
+Okta.
+
+This guide is an overview of building your own user interface. For information on the redirect model, see <StackSnippet snippet="redirectquickstart" inline />.
+
+## Sign-in flow
```

Comment:
Sign-in flow is the general term. There may be multiple challenge and enrollments in one sign-in flow (e.g. signing on after adding a new required authenticator means enrolling in that authenticator plus answering any other challenges for previously enrolled factors.)

Reply:
OK, let it stand then. I need to note that down somewhere and possibly add it to some other docs. 

As to the nav bar, I only meant changing the name of the H2 at this point.  The (Self-)Registration flow incorporates a sign-in too so that would be separate but inextricably tied to this work.